bot/utils/parameter_value_generator.py

```diff
@@ -12,12 +11,9 @@ def __init__(self, fun, parameter):
         self.fun = fun
         self.parameter = parameter
 
-    def __call__(self, *args):
```

Comment:
why don't you need this anymore?

Reply:
Oh, this was a mistake:) I was testing some stuff out, if you look at #45 and #46, the coverage is going down because of this, and then I committed the removed `__call__` method by mistake. I will add this back, thanks!